ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/data/processed/ColorsFactory.kt

```diff
@@ -23,7 +22,7 @@ internal object ColorsFactory {
             text2 = Color(text2ColorInt),
             callToActionBackground = Color(callToActionBackgroundColorInt),
             callToActionForeground = Color(callToActionForegroundColorInt),
-            callToActionSecondaryBackground = Color(callToActionSecondaryBackgroundColorInt),
+            callToActionSecondaryBackground = callToActionSecondaryBackgroundColorInt?.let(::Color),
```

Comment:
Hmm do we need to change `callToActionSecondaryBackgroundColorInt` to not fallback to `callToActionBackgroundColorInt`? Otherwise, it will always have a value I think?

Reply:
Yup I changed that above